[2023-04-14T06:24:44.853+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.inject_data_Kharkiv manual__2023-04-14T06:24:32.426204+00:00 [queued]>
[2023-04-14T06:24:44.927+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.inject_data_Kharkiv manual__2023-04-14T06:24:32.426204+00:00 [queued]>
[2023-04-14T06:24:44.936+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T06:24:44.939+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-14T06:24:44.942+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T06:24:44.997+0000] {taskinstance.py:1309} INFO - Executing <Task(SqliteOperator): inject_data_Kharkiv> on 2023-04-14 06:24:32.426204+00:00
[2023-04-14T06:24:45.012+0000] {standard_task_runner.py:55} INFO - Started process 7717 to run task
[2023-04-14T06:24:45.052+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'inject_data_Kharkiv', 'manual__2023-04-14T06:24:32.426204+00:00', '--job-id', '502', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpip8u6ha_']
[2023-04-14T06:24:45.064+0000] {standard_task_runner.py:83} INFO - Job 502: Subtask inject_data_Kharkiv
[2023-04-14T06:24:45.263+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag.inject_data_Kharkiv manual__2023-04-14T06:24:32.426204+00:00 [running]> on host ffea11fb791c
[2023-04-14T06:24:45.504+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag
AIRFLOW_CTX_TASK_ID=inject_data_Kharkiv
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T06:24:32.426204+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-14T06:24:32.426204+00:00
[2023-04-14T06:24:45.508+0000] {sql.py:254} INFO - Executing: 
                INSERT INTO measures_ext (city, timestamp, temp, humidity, cloudiness, wind_speed) VALUES
                    ("Kharkiv",
                    1681453108,
                    280.8,
                    87,
                    "light rain",
                    4.99);
            
[2023-04-14T06:24:45.525+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-14T06:24:45.837+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-14T06:24:45.846+0000] {sql.py:375} INFO - Running statement: 
                INSERT INTO measures_ext (city, timestamp, temp, humidity, cloudiness, wind_speed) VALUES
                    ("Kharkiv",
                    1681453108,
                    280.8,
                    87,
                    "light rain",
                    4.99);
            , parameters: None
[2023-04-14T06:24:45.877+0000] {sql.py:384} INFO - Rows affected: 1
[2023-04-14T06:24:45.924+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=inject_data_Kharkiv, execution_date=20230414T062432, start_date=20230414T062444, end_date=20230414T062445
[2023-04-14T06:24:46.014+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-14T06:24:46.056+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
